teams/rustdoc-contributors.toml

```
@@ -0,0 +1,22 @@
+name = "rustdoc-contributors"
+subteam-of = "rustdoc"
+
+[people]
+leads = []
+members = [
+    "fmease"
+]
+
+[permissions]
+bors.rust.review = true
+dev-desktop = true
+perf = true
+
+[[github]]
+orgs = ["rust-lang", "rust-lang-nursery"]
+extra-teams = ["rustdoc"]
+
+[website]
+name = "Rustdoc team contributors"
+description = "Contributing to rustdoc on a regular basis"
```

```suggestion
description = "Contributing to Rustdoc on a regular basis"
```

It's also capitalized in `rustdoc.toml`

https://github.com/rust-lang/team/blob/b1b37232eb3a91d75de4168c53d882653c4ba7a6/teams/rustdoc.toml#L38

---

:+1: